components/create/CreateLanding.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="container">
+    <h1 class="title is-size-2">{{ $t('mint.landing.heading') }}</h1>
+
+    <div v-if="select === 'landing'" class="pack">
```

Comment:
I would use `columns is-mobile` ([Bulma doc](https://bulma.io/documentation/columns/responsiveness/#mobile-columns))